bootstrap/lib/cache.js

```diff
@@ -14,6 +15,14 @@ const parseBool = (value) => {
   return (_val === 'true' || _val === 'false') && _val === 'true'
 }
 
+// We generate an UUID per instance
+const uuid = () => {
```

Comment:
This is already used in the logging module, we should extract this function out and share the implementation